src/reportSources/legacyFflogs/__tests__/__snapshots__/eventAdapter.ts.snap

```diff
@@ -83,19 +83,25 @@ exports[`Event adapter individual events adapts calculateddamage 1`] = `
 Array [
```

Comment:
doesn't look like there's any `execute`s coming through here - something funky?

Reply:
Oh, might be because both of the example events happen to be statuses, haven't checked. Probably worth testing all 3 code paths for that.